cpp/include/cuml/fil/fil.h

```diff
@@ -116,7 +125,7 @@ struct treelite_params_t {
  */
 // TODO (canonizer): use std::variant<forest_t<float> forest_t<double>>* for pforest
```

Comment:
Can this comment be removed now?

Reply:
Done.

I've also updated some other documentation in this file, so that it now says that both `float` and `double` are supported.